docs/overview/running_project_windows.md

```diff
@@ -0,0 +1,54 @@
+# How to get the code running on Windows for testing
+There are some different steps to follow in order to run the e-mission server and the e-mission phone app in Windows from the command prompt. In this document, we will highlight the steps in the main documentation page (https://github.com/e-mission/e-mission-server#dependencies) where there are different steps to perform.
+
+Before start following the steps above, the first thing you want to do is to create this path in your computer:
+```
+C:\var\tmp
+```
+## e-mission server
+### Python distribution
+After installing and configuring Anaconda, there is the following step "*Setup the emission environment*". Instead of running the command stated in this step, please run the following commands into your command prompt:
+
+```
+conda env update --name emission --file setup/environment36.yml
+activate emission
+pip install six --upgrade
+```
+
+Note that every time you need to run the code, you have to run the command ```activate emission```. Also, note that you must be at the root of the project in order to access the directory ```setup/``` in the first command stated before.
+
+### Development
+In the development section, we will change the step 3 "*Start the server*". Instead of doing that, first, you have to create or update the following environment variable on windows:
+
+- Name of the variable: PYTHONPATH
+- Value of the variable: C:\MyPath\Anaconda3;C:\MyPath\e-mission-server
+
+Note that in the field "*Value of the variable*" there are two paths separated by a semicolon. The first one is the path on your computer to the folder in which you installed Anaconda. The second is the path in your computer where you have the root folder of the server code.
+
+After doing so, you have to copy the e-mission.bat script (located at scripts/overview/running_project_windows) into the root of your project and run the following command:
```

Comment:
I would be happy to have `e-mission.bat` checked in to the server repository directly. That way, users don't need to copy it around.

There actually used to be an `e-mission.bat` before, but nobody was using it and it bit-rotted, so I removed it.